### PR TITLE
SE rewrite

### PR DESCRIPTION
**Pull request by andreas-junghanns**

### CODE REVIEW

File: docs/5_2_scheduled_execution_api.adoc [hunk 1]
```diff
@@ -69,24 +54,24 @@ a|When an input Clock latexmath:[k_i] ticks, activate the corresponding model pa
...
-_The reason is to avoid different interpretations of the caching, since contrary to <<fmi-for-model-exchange, `FMI for Model Exchange`>>, <<fmi3ActivateModelPartition>> will perform the actual calculation instead of <<get-and-set-variable-values,`fmi3Get{VariableTypeExclClock}`>>, and therefore, dummy algebraic loops at communication points cannot be handled by an appropriate sequence of <<get-and-set-variable-values,`fmi3Get{VariableTypeExclClock}`>> and <<get-and-set-variable-values,`fmi3Set{VariableTypeExclClock}`>> calls as for Model Exchange.]_
 +
-Set/get calls before and after <<fmi3ActivateModelPartition>> are only allowed for variables that define a reference to the <<inputClock>> by the <<clocks>> attribute or variables that are associated to no <<Clock>>.
-_[Note, that to avoid data inconsistencies and safeguard predictable behavior with <<get-and-set-variable-values,`fmi3Get{VariableType}`>>, <<get-and-set-variable-values,`fmi3Set{VariableType}`>> a unique assignment of the respective variables to model partitions via its associated <<Clock>> is strongly recommended._
```
Comment (MBlesken) [inline docs/5_2_scheduled_execution_api.adoc:88]: Not okay: This information is important and should be mentioned here.

File: docs/5_1_scheduled_execution_math.adoc [hunk 2]
```diff
@@ -5,25 +5,12 @@
 In Scheduled Execution the importer has to ensure that model partitions are scheduled according to their activation time and priorities.
 This is realized in <<ClockActivationMode>> by calling <<fmi3ActivateModelPartition>> triggering their respective Clocks.
 
-After <<fmi3ActivateModelPartition>> has been called for a <<fixed-clock,`fixed`>>, <<tunable-clock,`tunable`>> or <<changing-aperiodic-clock,aperiodic>> Clock the FMU provides the information on when the Clock will tick again, i.e. when the corresponding model partition must be scheduled the next time.
```
Comment (MBlesken) [inline docs/5_1_scheduled_execution_math.adoc:8]: Why is this deleted?

File: docs/5_1_scheduled_execution_math.adoc [hunk 3]
```diff
@@ -5,25 +5,12 @@
...
-To retrieve this information the importer calls <<fmi3GetInterval>> on the Clock.
-
 The FMU may request to schedule another model partition even while currently a model partition is being executed.
 
-A new interval for a <<countdown-aperiodic-clock,`countdown`>> Clock can be provided at any time during the execution of any model partition.
```
Comment (MBlesken) [inline docs/5_1_scheduled_execution_math.adoc:13]: Why is this deleted?

File: docs/5_1_scheduled_execution_math.adoc [hunk 4]
```diff
@@ -5,25 +5,12 @@
...
-The FMU signals that a <<countdown-aperiodic-clock,`countdown`>> Clock is about to tick by calling <<fmi3ClockUpdateCallback>>.
-The importer calls <<fmi3GetInterval>> for all <<countdown-aperiodic-clock,`countdown`>> Clocks.
-If a new interval latexmath:[T_{\mathit{interval, i}}] is provided, i.e. return argument <<qualifiers>> = <<fmi3IntervalChanged>>, the importer initiates the scheduling of the associated model partitions regarding the delay of the returned interval.
-
 In case more than one Clock ticks at the same time instant, the scheduler needs a priority to define the activation sequence of the associated model partitions.
 This ordering is defined by the <<priority>> attributes of the Clock.
 For Scheduled Execution this attribute is mandatory for every Clock.
 
-_[A clock's <<priority>> information is required to support the scheduled execution of several model partitions._
```
Comment (MBlesken) [inline docs/5_1_scheduled_execution_math.adoc:22]: Why is this deleted? Or is it moved? Hard do figure out in such a huge PR ;)

File: docs/5_2_scheduled_execution_api.adoc (hunk 1 above)
Comment (MBlesken) [reply docs/5_2_scheduled_execution_api.adoc:88]: ...and refer to [[textExample]] ;)

**Review by MBlesken — CHANGES_REQUESTED** [review]
Why are these parts removed from SE?

File: docs/5_1_scheduled_execution_math.adoc (hunk 2 above)
Comment (andreas-junghanns) [reply docs/5_1_scheduled_execution_math.adoc:8]: That is already described in the clocks section.

File: docs/5_1_scheduled_execution_math.adoc (hunk 3 above)
Comment (andreas-junghanns) [reply docs/5_1_scheduled_execution_math.adoc:13]: see above

File: docs/5_1_scheduled_execution_math.adoc (hunk 4 above)
Comment (andreas-junghanns) [reply docs/5_1_scheduled_execution_math.adoc:22]: See Clocks section - there it is complete and correct - we wanted to remove the redundancy with slight differences.
